### DISCUSSION TITLE
Notifications bar overlaps new commit prompt

### DISCUSSION THREAD

**Discussion by adamlui** [community/community#57017] [Repositories]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/10906554/2e21ca89-0902-44c8-913b-6769969c0563)

The prompt's z-index needs to be higher

**Comment by DSDmark** [comment]
Hi @adamlui,

I'm not sure, it is a bug. In my case, it's working fine. If you are facing that issue. Then, it is helpful to add some more details like which browser, OS you are using when you're facing that issue.

**Reply by adamlui** [reply]
@DSDmark You're unsure it's a bug because it works for you? Or you think it's intentional for notifications bar to overlap but not top bar? (Bug means unintended behavior for any user btw, not just you)
